Thirdparty/g2o/g2o/core/estimate_propagator.h

```diff
@@ -135,7 +130,7 @@ namespace g2o {
           size_t operator ()(const OptimizableGraph::Vertex* v) const { return v->id();}
       };
 
-      typedef std::tr1::unordered_map<OptimizableGraph::Vertex*, AdjacencyMapEntry, VertexIDHashFunction> AdjacencyMap;
+      typedef std::unordered_map<OptimizableGraph::Vertex*, AdjacencyMapEntry, VertexIDHashFunction> AdjacencyMap;
```

Comment:
It sounds like most things from `tr1` got included in the standard library in C++11, so this should no longer be necessary.